### DISCUSSION TITLE
Has there been a change to disallow clip-path ?

### DISCUSSION THREAD

**Discussion by JohnSmith-LT** [cssbattle/forum#321] [Bugs reports]
Hi,

Any time that I submit a solution that has a clip-path in it now gives the notification when I submit:

**You scored 0 with 0% match**

![image](https://user-images.githubusercontent.com/3907362/201437865-b9daa890-73a6-4154-8000-4feccf29e4fa.png)

Even the open solutions such as https://cssbattle.dev/play/120 that contain clip-path now give that message in both Chrome (latest) and in Microsoft Edge (latest)...

Is this down to some change that has been made or is it a bug or is anyone else seeing this?

For info I have a Pro account

John

**Comment by chinchang** [comment]
Yes, the encoding hack that was used to put restricted SVG in a `clip-path` is now prevented.
https://cssbattle.dev/changelog

**Comment by JohnSmith-LT** [comment]
Thanks @chinchang I see that now...

**Comment by Beowolve** [comment]
You can change to polygon, to make that basic method work. However its way longer and looks ugly that way. A version without polygon will be shorter for sure but this is how you would transform it.

## 281 chars - polygon
`<body bgcolor=0>▜▛ ▔▗▖<a>▖▖</a>    ▟▙<p>█<style>html{color:54C144;border:solid+5vw}a{color:fff}*+*{font:5vw/25%a;margin:52%77%0%40}p{color:C74E4E;margin:-402%0;zoom:21.2;clip-path:polygon(0 0,0 6q,10q 6q,10q 4q,0 4q,0 2q,4q 2q,4q 0,6q 0,6q 2q,10q 2q,10q 0,12q 0,12q 9q,14q 9q,14q 0`

But u could also make this just box shadow without unicode and get shorter already
## 280 chars - box shadow version
`<img><a><style>img,html{border:solid 5vw#54C144;background:#000}body{border-top:double 30vw#C74E4E;margin:40 120 40 40;box-shadow:5ch 0,5em 0#C74E4E}img{margin:20 20-17 0;box-shadow:5em -35vw}a{border-right:32q double#fff;font:58%a;box-shadow:-90q 0 0+5q,-42q 0#54C144,-27q 0 0+5q`

**Comment by JohnSmith-LT** [comment]
Thanks @Beowolve. I had a few clip-path solutions queued up from 15th since I thought my 3000 tabs being open was causing some kind of submit error in Chrome...but after seeing this I have reworked many of them. Some to polygon but also some to inline clip-path which still appears to be allowed... for now... 😉
